internal/eval/eval_test.go

```diff
@@ -462,6 +463,117 @@ func testEvalJSONARRLEN(t *testing.T, store *dstore.Store) {
 	runEvalTests(t, tests, evalJSONARRLEN, store)
 }
 
+func testEvalJSONOBJLEN(t *testing.T, store *dstore.Store) {
```

Comment:
Add test cases that validate behaviour of objlen on incompatible fields (integers, arrays, strings)

Reply:
added in this [commit](https://github.com/DiceDB/dice/pull/593/commits/54139be1a471092a6f00e0cda24b0ac225da38d3)